### ISSUE TITLE
/lib/misc.zsh:3: parse error near `then'

### ISSUE THREAD

**Issue by naartjie** [robbyrussell/oh-my-zsh#4004]
I just pulled the latest master 8921902388b, and I am getting a

```sh
/Users/marchi/.oh-my-zsh/lib/misc.zsh:3: parse error near `then'
```
I did a `git pull` manually. Did I need to do something else?

**Comment by dmxcsnsbh** [comment]
`if [[ -e "$d/url-quote-magic" ]]; then`

There must be a space follow `magic"`

**Comment by naartjie** [comment]
Ah, thanks, that fixes it. Should I PR?

**Comment by ncanceill** [comment]
Please NOTE this is fixed in #4002 already! no need to submit your own PR.

**Comment by naartjie** [comment]
Okay, closing this one then. Thanks!!
